Test/CoreSDK.Test/Shared/DataContracts/PerformanceCounterTelemetryTest.cs

```diff
@@ -2,8 +2,9 @@
 {
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
```

Comment:
This was compiled with Release. I assume System namespaces should precede AI namespaces? Not sure why Stylecop did not flag this... Will fix